openverse_api/catalog/api/examples/audio_requests.py

```diff
@@ -29,9 +29,9 @@
 )
 
 audio_search_curl = f"""
-# Search for music titled "Friend" by Rob Costlow
-curl {auth} "{origin}/v1/audio/?title=Friend&creator=Rob%20Costlow"
-"""
+# Search for music titled "Wish You Were Here" by The.madpix.project
+curl {auth} "{origin}/v1/audio/?title=Wish%20You%20Were%20Here&creator=The.madpix.project"
+"""  # noqa
```

Comment:
Out of curiosity, what's being suppressed here? (maybe this could be a `noqa: <error>` directive?)

Reply:
The length of the line